src/gigs-board/pages/community/Telegram.jsx

```diff
@@ -0,0 +1,134 @@
+    linkProps.nearDevGovGigsWidgetsAccountId =
+      props.nearDevGovGigsWidgetsAccountId;
+  }
+  if (props.referral) {
+    linkProps.referral = props.referral;
+  }
+  const linkPropsQuery = Object.entries(linkProps)
+    .map(([key, value]) => `${key}=${value}`)
+    .join("&");
+  return `#/${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.pages.${widgetName}${
+    linkPropsQuery ? "?" : ""
+  }${linkPropsQuery}`;
+}
+/* END_INCLUDE: "common.jsx" */
+
+/* INCLUDE: "communities.jsx" */
+const communities = {
+  "zero-knowledge": {
+    overviewId: 397,
+    eventsId: 401,
+    icon: "https://ipfs.near.social/ipfs/bafkreiajwq6ep3n7veddozji2djv5vviyisabhycbweslvpwhsoyuzcwi4",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreihgxg5kwts2juldaeasveyuddkm6tcabmrat2aaq5u6uyljtyt7lu",
+    title: "Zero Knowledge",
+    desc: "Building a zero knowledge ecosystem on NEAR.",
+    telegram: "NearZeroKnowledge",
+  },
+  protocol: {
+    overviewId: 412,
+    eventsId: 413,
+    icon: "https://ipfs.near.social/ipfs/bafkreidpitdafcnhkp4uyomacypdgqvxr35jtfnbxa5s6crby7qjk2nv5a",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreicg4svzfz5nvllomsahndgm7u62za4sib4mmbygxzhpcl4htqwr4a",
+    title: "Protocol",
+    desc: "Supporting the ongoing innovation of the NEAR Protocol.",
+    telegram: "NEAR_Protocol_Community_Group",
+  },
+  tooling: {
+    overviewId: 416,
+    eventsId: 417,
+    icon: "https://ipfs.near.social/ipfs/bafkreie2eaj5czmpfe6pe53kojzcspgozebdsonffwvbxtpuipnwahybvi",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreiehzr7z2fhoqqmkt3z667wubccbch6sqtsnvd6msodyzpnf72cszy",
+    title: "Tooling",
+    desc: "Supporting the ongoing innovation of tooling.",
+    telegram: "NEAR_Tools_Community_Group",
+  },
+  "contract-standards": {
+    overviewId: 414,
+    eventsId: 415,
+    icon: "https://ipfs.near.social/ipfs/bafkreiepgdnu7soc6xgbyd4adicbf3eyxiiwqawn6tguaix6aklfpir634",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreiaowjqxds24fwcliyriintjd4ucciprii2rdxjmxgi7f5dmzuscey",
+    title: "Contract Standards",
+    desc: "Coordinating the contribution to the NEAR dapp standards.",
+    telegram: "nearnft",
+  },
+};
+/* END_INCLUDE: "communities.jsx" */
+
+if (!props.label) {
+  return (
+    <div class="alert alert-danger" role="alert">
+      Error: label is required
+    </div>
+  );
+}
+
+const community = communities[props.label];
+
+const group = community.telegram;
+
+const groupInfo = fetch(
+  `https://j96g3uepe0.execute-api.us-east-1.amazonaws.com/groups/${group}`
+);
+if (groupInfo === null || !groupInfo.ok) {
+  return "Loading ...";
+}
+
+const messageIds = groupInfo.body.messageIds;
+
+const Telegram = (
+  <div>
+    <iframe
+      iframeResizer={{ log: true }}
+      src={
+        "https://j96g3uepe0.execute-api.us-east-1.amazonaws.com/groups-ui/" +
+        group
+      }
+      frameborder="0"
+      style={{
+        width: "1px",
+        minWidth: "100%",
```

Comment:
Can you leave a comment about this quirk? Do we still need it, by the way?

Reply:
Yes, in this non-nested case, iframe resizer works, with these required by iframe-resizer